src/lib/utils.ts

```diff
@@ -28,6 +28,19 @@ export const setDate = <T>(input: T, date: Date = new Date()) => {
   }
 }
 
+export const getStreetNumber = ({reg_number, building_number}) => {
+  if (reg_number && building_number){
```

Comment:
Cislo asi nemoze byt `0`, ze? 😆 Napr. `123/0`. 

Ak nie, tak potom je to dobre ako je 😄

Reply:
ca1e2f2fe82a8d9c5dc04121c62d97192e57bcd0

a bit of a stretch 😅  ale ako sa hovori Slovensko krajina nekonecnych edge casov, popravde netusim ci tam moze byt `0` 🤷🏻 ale teraz uz moze 😆